addons/mail/static/src/core/common/out_of_focus_service.js

```diff
@@ -143,7 +143,7 @@ export class OutOfFocusService {
     }
 
     _playSound() {
-        if (this.canPlayAudio && this.multiTab.isOnMainTab()) {
+        if (this.canPlayAudio && this.multiTab.isOnMainTab() && this.store.settings.messageSound) {
             this.soundEffectService.play("new-message");
```

Comment:
Not because of this PR, but there is an inconsistency between play sound with a notif (in general terms) and playing specifically the message sound in that case. It looks like this should have been in new message from the start.

Reply:
The service is poorly named, all it does is conditionally determine whether new message should put sound or change tab title.